doc/apimanager-reference.md

```diff
@@ -21,7 +21,7 @@ This resource is the resource used to deploy a 3scale API Management solution.
 
 | **Field** | **json/yaml field**| **Type** | **Required** | **Default value** | **Description** |
 | --- | --- | --- | --- | --- | --- |
-| ProductVersion | `productVersion` | ProductVersion | Yes | N/A | 3Scale API Management solution release version. Currently the values "2.4" and "upstream" are supported. **Warning**, "upstream" uses 3scale unstable nightly images and it is intended only for development purposes and to be used in non-productive environments |
+| ProductVersion | `productVersion` | ProductVersion | Yes | N/A | 3Scale API Management solution release version. Currently the values "2.5" and "upstream" are supported. **Warning**, "upstream" uses 3scale unstable nightly images and it is intended only for development purposes and to be used in non-productive environments |
```

Comment:
2.4 is not supported? I cannot deploy 2.4 with the operator?

Reply:
I'd say no, because the operator is part of the 2.5 release and some needed APIs at Porta level have been added for the 2.5 release that are needed for the capabilities part.

Reply:
Let me clarify... you could deploy the APIManager in a standalone way with 2.4 with the operator but we cannot guarantee it would not be used with the capabilities part, and operator is not part of the 2.4 release so I don't see the need to include it as an additional version to support.

Reply:
In fact, in another PR I planned to remove the 2_4_release code but I've not done so because it's useful to use to have it for testing purposes until 2.5 is released

Reply:
ok